### DISCUSSION TITLE
【建议】增加disqus评论插件

### DISCUSSION THREAD

**Discussion by Bashir-1** [tangly1024/NotionNext#2100] [Ideas]
**为什么提出这个新的特性改动**
disqus

**描述一下你推荐的解决方案**
[disqus](https://disqus.com/)评论插件功能更加强大，配置简单

**描述一下你考虑过的其它替代解决方案**

**补充说明**

**Comment by JIeJaitt** [comment]
跟一个，希望早日支持`disqus`
